packages/bun-types/bun.d.ts

```diff
@@ -3071,7 +3075,7 @@ declare module "bun" {
      *
      * If the process hasn't exited yet, this will return `null`
      */
-    readonly exitCode: number | null;
+    readonly exitCode: number | DOMException | null;
```

Comment:
It shouldn't return a DOMException

Reply:
should we add some other way of knowing if the request was killed or aborted?